committee/committee.go

```diff
@@ -243,6 +235,27 @@ func (ec *committee) Status() STATUS {
 	}
 }
 
+func (ec *committee) SyncInSplit(tipHeight uint64) error {
+	zap.L().Info("new ethereum block", zap.Uint64("height", tipHeight))
+	fetchStart := ec.nextHeight + ec.interval
+	if tipHeight < fetchStart {
+		return nil
+	}
+	fetchHeight := uint64(ec.fetchInParallel) * ec.interval
+	for ec.currentHeight < tipHeight {
+		if ec.nextHeight+fetchHeight < tipHeight {
+			ec.currentHeight = ec.nextHeight + fetchHeight
+		} else {
+			ec.currentHeight = tipHeight
+		}
+		err := ec.Sync(ec.currentHeight)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 func (ec *committee) Sync(tipHeight uint64) error {
```

Comment:
Sync is only called by SyncInSplit. Would you please consolidate the this simple method's statements into SyncInSplit directly?

Reply:
sure